### PR TITLE
[DONE]ansible raffler

### PR DESCRIPTION
**Pull request by BauRo**
This is a ansible raffler with a little bit of work still to do.
the-right-tools-for-the-job

### CODE REVIEW

File: bauro-ansible/host_vars/localhost.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+# file: host_vars/localhost
+
+ansible_connection: local
+
+names_file: ../example_names
```
Comment (lucasvanlierop) [inline bauro-ansible/host_vars/localhost.yml]: Can you make the this a command line argument for `run.sh`, so we can use a given file?

**Comment by lucasvanlierop** [comment]
@erikaheidi any comments maybe? :-)

File: bauro-ansible/run.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```
Comment (ramondelafuente) [inline bauro-ansible/run.sh]: If you add something like `--extra_vars "names_file=$1"` you don't need the variable in host_vars.

Comment (BauRo) [inline bauro-ansible/run.sh]: Thanxs, added. :-)

File: bauro-ansible/host_vars/localhost.yml (hunk 1 above)
Comment (BauRo) [inline bauro-ansible/host_vars/localhost.yml]: argument possibility added

**Comment by BauRo** [comment]
Added argument option
funny-arabic-rich-cheetah

**Comment by ramondelafuente** [comment]
Looks good to go :+1: 

fixed

**Comment by rosstuck** [comment]
My god.

**Comment by lucasvanlierop** [comment]
The parameter works!

![http://cdn.meme.am/instances/58694934.jpg] (http://cdn.meme.am/instances/58694934.jpg)

**Comment by erikaheidi** [comment]
slow clap

**Pull request merged**